src/styles/components/tooltip/styles.less

```diff
@@ -1,14 +1,18 @@
 & when (@tooltip-enabled) {
 
   .tooltip {
+    will-change: opacity, transform, visibility;
     z-index: @z-index-tooltip;
 
     .tooltip-content {
       background: @tooltip-background-color;
       border-radius: @tooltip-border-radius;
       color: @white;
       font-size: @tooltip-font-size;
+      -moz-osx-font-smoothing: grayscale;
```

Comment:
Let's add a comment for this, so we know why it is here, when we need to change it (when it gets added as a rule in autoprefixer)